pages/common/b2sum.md

```diff
@@ -11,10 +11,18 @@
 
 `b2sum {{path/to/file1}} {{path/to/file2}}`
 
+- Calculate the BLAKE2 checksum from stdin:
+
+`{{some_command}} | b2sum`
+
 - Read a file of BLAKE2 sums and filenames and verify all files have matching checksums:
 
-`b2sum -c {{path/to/file.b2}}`
+`b2sum --check {{path/to/file.b2}}`
 
-- Calculate the BLAKE2 checksum from stdin:
+- Only show a message for files for which verification fails or missing files:
```

Comment:
I think we can reword this .. but I am unable to come-up with a rephrase.
The thing that seems odd to me is the ` or missing files`.
But, on the other hand, it gets the meaning conveyed .. so we could keep it.
Thoughts ?

Reply:
**Only show a message for files when verification fails or a file is missing:**        ?

Reply:
**Only show a message for missing files and when verification fails**

Reply:
I think the latter sounds good.
This to be specific:
```Only show a message for missing files or when verification fails```